azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -29,6 +30,17 @@ class MQTTTransportStage(PipelineStage):
     is not in the MQTT group of operations, but can only be run at the protocol level.
     """
 
+    def __init__(self):
+        super(MQTTTransportStage, self).__init__()
+
+        # The sas_token will be set when Connetion Args are received
+        self.sas_token = None
```

Comment:
>sas_token [](start = 13, length = 9)

observation with no action required: we are getting better at marking methods as convention private, but we almost never mark attributes this way (or at least I don't). Are convention private attributes on internal objects a practice we should encourage?  My gut says yes.